flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionVertexInputConstraintTest.java

```diff
@@ -106,7 +106,6 @@ public void testInputConstraintANY() throws Exception {
 		eg.start(mainThreadExecutor);
 		eg.scheduleForExecution();
 
-
```

Comment:
Unrelated changes?

Reply:
That's right. Let me move it into a hotfix.

Reply:
done.

Reply:
Interesting, why not this checkstyle violations complain before?

Reply:
Some flink-runtime classes style checks are suppressed for now because there were lots of violations when the checkstyle was introduced.
It should be good to fix them by the way, so that one day in the future we can remove those checkstyle suppression.

Reply:
we only wanted to make these changes though when we touch a file for other changes though; the only reason why we aren't applying checkstyle everywhere is the potential for merge conflicts; if we now go around and randomly apply checkstyle we still have the same risk.